app/src/views/FieldReportForm/ContextFields/TitlePreview/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { useMemo } from 'react';
+import { TextOutput } from '@ifrc-go/ui';
+import { useTranslation } from '@ifrc-go/ui/hooks';
+
+import useAlert from '#hooks/useAlert';
+import { useRequest } from '#utils/restRequest';
+
+import i18n from './i18n.json';
+
+interface Props {
+    country: number;
+    disasterType: number;
+    event?: number;
+    isCovidReport?: boolean;
+    startDate?: string;
+    title: string;
+}
+
+function TitlePreview(props: Props) {
+    const {
+        country,
+        disasterType,
+        event,
+        isCovidReport,
+        startDate,
+        title,
+    } = props;
+
+    const strings = useTranslation(i18n);
+    const alert = useAlert();
+
+    const variables = useMemo(() => ({
+        countries: [country],
+        is_covid_report: isCovidReport,
+        start_date: startDate,
+        dtype: disasterType,
+        event,
+        title,
+    }), [country, isCovidReport, startDate, disasterType, event, title]);
```

Comment:
lets debounce the variables to avoid too many requests

Reply:
Done in the latest commit